fsspec/parquet.py

```diff
@@ -78,11 +98,20 @@ def open_parquet_file(
     if fs is None:
         fs = url_to_fs(path, storage_options=(storage_options or {}))[0]
 
+    # For now, `columns == []` not supported. Just use
+    # default `open` command with `path` input
+    if columns is not None and len(columns) == 0:
```

Comment:
`if not columns` ?

Reply:
Well, we don't want to do this if `columns=None`, only if `columns=[]` I guess. Perhaps we can just assume columns will be a list

Reply:
sorry, misread